utils/readmp.f90

```diff
@@ -0,0 +1,46 @@
+!> @file
+!> @brief Read BUFR file containing embedded DX tables, and print
+!> each report one at a time
```

Comment:
Author and date tags?

Reply:
The version control system keeps track of this information... there really shouldn't be a need to have to annotate each file this way.

Reply:
Please add author tags to all code in all NCEPLIBS projects. Date tags are optional but encouraged.

The repo is not sufficient for this purpose, as evidenced by the fact that I have over 200K lines of Fortran/C code, some 30+ years old, all added to GitHub recently, with incorrect attribution. Had those programmers not properly documented their work, we would now have no idea who wrote what. 

When a scientist publishes a paper, their name goes at the top, which documents their involvement, gives them proper credit, and encourages them to take extra care that the paper is correct, as it reflects directly on them. 

For the same reasons, all NCEPLIBS codes will contain proper attribution.

Reply:
done per the decree.